docs/admin/installation/hardware.rst

```diff
@@ -619,12 +569,12 @@ removed the wireless components with ease. It's possible to re-purpose old
 laptops from other manufacturers, as long as the wireless components are
 removable.
 
-Just as with the servers, you can also use an Intel NUC for the *SVS*. As noted
+Just as with the servers, you can also use a NUC for the *SVS*. As noted
 before, NUCs do not ship with a hard drive, and older models can be configured
 without any wireless components. However, NUCs *do* contain an IR receiver,
 which we recommend taping over with opaque masking tape.
 
-If you choose to use an Intel NUC, you must use an older model that offers wireless
+If you choose to use a NUC, you must use an older model that offers wireless
```

Comment:
I would say "a model that offers" rather than "an older model that offers"...

Reply:
Done